app/models/TuningParameterConstraint.java

```diff
@@ -0,0 +1,109 @@
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.JoinTable;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+
+import com.avaje.ebean.annotation.UpdatedTimestamp;
+
+import play.db.ebean.Model;
+
+
+
+//  job_definition_id int(10) unsigned NOT NULL COMMENT 'foreign key from job_definition table',
+//          constraint_type enum('BOUNDARY', 'INTERDEPENDENT') NOT NULL COMMENT 'Constraint type',
+//        tuning_parameter_id int(10) unsigned NULL COMMENT 'foreign key from tuning_parameter table',
+//        lower_bound int(10) unsigned NOT NULL,
+//        upper_bound int(10) unsigned NOT NULL,
+//        created_ts timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP,
+//        updated_ts timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP ,
+//        PRIMARY KEY(job_definition_id),
+//        CONSTRAINT param_constraints_f1 FOREIGN KEY (job_definition_id) REFERENCES job_definition (id),
+//        CONSTRAINT param_constraints_f2 FOREIGN KEY (tuning_parameter_id) REFERENCES tuning_parameter (id)
+
+@Entity
+@Table(name = "tuning_parameter_constraint")
+public class TuningParameterConstraint extends Model {
+
+  private static final long serialVersionUID = 1L;
+
+  public enum ConstraintType {
+    BOUNDARY, INTERDEPENDENT
+  }
+
+  public static class TABLE {
+    public static final String TABLE_NAME = "tuning_parameter_constraint";
+    public static final String id = "id";
+    public static final String jobDefinitionId = "jobDefinitionId";
+    public static final String constraintType = "constraintType";
+    public static final String constraintId = "constraintId";
+    public static final String tuningParameterId = "tuningParameterId";
+    public static final String lowerBound = "lowerBound";
+    public static final String upperBound = "upperBound";
+    public static final String createdTs = "createdTs";
+    public static final String updatedTs = "updatedTs";
+  }
+
+
+  @Id
+  @GeneratedValue(strategy = GenerationType.IDENTITY)
+  public Integer id;
+
+  @ManyToOne(cascade = CascadeType.ALL)
+  @JoinTable(name = "job_definition", joinColumns = {@JoinColumn(name = "job_definition_id", referencedColumnName = "id")})
+  public JobDefinition jobDefinition;
+
+  @Enumerated(EnumType.STRING)
+  @Column(nullable = false)
+  public ConstraintType constraintType;
+
+  @Column(nullable = false)
+  public Integer constraintId;
+
+  @ManyToOne(cascade = CascadeType.ALL)
+  @JoinTable(name = "tuning_parameter", joinColumns = {@JoinColumn(name = "tuning_parameter_id", referencedColumnName = "id")})
+  public TuningParameter tuningParameter;
+
+  @Column(nullable = false)
+  public Double lowerBound;
+
+  @Column(nullable = false)
+  public Double upperBound;
+
+  @Column(nullable = false)
+  public Timestamp createdTs;
+
+  @Column(nullable = false)
+  @UpdatedTimestamp
+  public Timestamp updatedTs;
+
```

Comment:
Can you override the save() and update() as done in other models. The reason being, ebean does not updates the updatedTs by default nor does the db (the reason being H2 db does not support "on update currenttimestamp).

Reply:
added